lib/decorators/deadline/get-deadline.js

```diff
@@ -9,6 +9,8 @@ const EXTENDED_DEADLINE = 55;
 // configure bank holidays
 moment.updateLocale('en', { holidays: bankHolidays });
 
+const daysSinceDate = (date, from = new Date()) => moment(from).diff(moment(date), 'days');
```

Comment:
If this is never called with a second argument then it's probably a lot easier to omit it completely
```suggestion
const daysSinceDate = (date) => moment().diff(moment(date), 'days');
```

Reply:
But also isn't this intended to be based on the activity date, not the current date?

Reply:
The decorator does not run against the task data stored in `activity_log.event.data`, so this is only useful for the current deadline.

We'll still have to calculate the `daysSinceDeadline` on the UI side for activity log entries, but we'll use the deadline stored against the activity rather than the current deadline (fix in another PR).